### ISSUE TITLE
table refinery_pages has no column named slug

### ISSUE THREAD

**Issue by ciponthenet** [refinery/refinerycms#3315]
With Refinery 4.0.0
running 

> refinerycms rickrockstar

fails with error:

> rake aborted!
> 
> ActiveRecord::StatementInvalid: SQLite3::SQLException: table refinery_pages has no column named slug: INSERT INTO "refinery_pages" ("slug", "link_url", "menu_match", "deletable", "lft", "rgt", "created_at", "updated_at") VALUES (?, ?, ?, ?, ?, ?, ?, ?)
> 
> ...

**Comment by bricesanchez** [comment]
Hi! Are you trying to update an old Refinery CMS?

**Comment by bricesanchez** [comment]
I'm not able to reproduce your error.

If you attempt to upgrade an old Refinery, be sure you have installed all new migrations : 
`rake railties:install:migrations` and `rake db:migrate`

**Comment by ciponthenet** [comment]
I actually tried it with a fresh ruby 2.4.2 installation and a newly created gemset.

**Comment by bricesanchez** [comment]
Is it working if you follow this guide: http://www.refinerycms.com/guides/getting-started ?

**Comment by ciponthenet** [comment]
Yes.
And manually running `rake db:seed` after the error fixes the refinery app.

**Comment by bricesanchez** [comment]
Could you copy/paste your stacktrace ?

**Comment by ciponthenet** [comment]
Happens after the migrations. Here it goes:

```
rake aborted!
ActiveRecord::StatementInvalid: SQLite3::SQLException: table refinery_pages has no column named slug: INSERT INTO "refinery_pages" ("slug", "link_url", "menu_match", "deletable", "lft", "rgt", "created_at", "updated_at") VALUES (?, ?, ?, ?, ?, ?, ?, ?)
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/sqlite3-1.3.13/lib/sqlite3/database.rb:91:in `initialize'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/sqlite3-1.3.13/lib/sqlite3/database.rb:91:in `new'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/sqlite3-1.3.13/lib/sqlite3/database.rb:91:in `prepare'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/sqlite3_adapter.rb:212:in `block (2 levels) in exec_query'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies/interlock.rb:46:in `block in permit_concurrent_loads'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/concurrency/share_lock.rb:185:in `yield_shares'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies/interlock.rb:45:in `permit_concurrent_loads'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/sqlite3_adapter.rb:209:in `block in exec_query'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract_adapter.rb:612:in `block (2 levels) in log'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract_adapter.rb:611:in `block in log'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/notifications/instrumenter.rb:21:in `instrument'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract_adapter.rb:603:in `log'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/sqlite3_adapter.rb:208:in `exec_query'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/database_statements.rb:102:in `exec_insert'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/database_statements.rb:133:in `insert'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/query_cache.rb:17:in `insert'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/relation.rb:61:in `insert'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/persistence.rb:589:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/counter_cache.rb:178:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/locking/optimistic.rb:76:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/attribute_methods/dirty.rb:296:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/callbacks.rb:340:in `block in _create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/callbacks.rb:131:in `run_callbacks'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/callbacks.rb:827:in `_run_create_callbacks'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/callbacks.rb:340:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/timestamp.rb:95:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/persistence.rb:563:in `create_or_update'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/callbacks.rb:336:in `block in create_or_update'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/callbacks.rb:131:in `run_callbacks'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/callbacks.rb:827:in `_run_save_callbacks'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/callbacks.rb:336:in `create_or_update'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/persistence.rb:162:in `save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/validations.rb:50:in `save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/attribute_methods/dirty.rb:43:in `save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/transactions.rb:313:in `block in save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/transactions.rb:384:in `block in with_transaction_returning_status'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/database_statements.rb:235:in `block in transaction'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/transaction.rb:194:in `block in within_new_transaction'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/transaction.rb:191:in `within_new_transaction'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/database_statements.rb:235:in `transaction'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/transactions.rb:210:in `transaction'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/transactions.rb:381:in `with_transaction_returning_status'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/transactions.rb:313:in `save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/suppressor.rb:46:in `save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/persistence.rb:51:in `create!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/refinerycms-pages-4.0.0/db/seeds.rb:2:in `<top (required)>'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `block in load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:258:in `load_dependency'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/railties-5.1.4/lib/rails/engine.rb:549:in `load_seed'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/railties-5.1.4/lib/rails/railtie.rb:185:in `public_send'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/railties-5.1.4/lib/rails/railtie.rb:185:in `method_missing'
/Users/e832319/work/ruby/rickrockstar/db/seeds.rb:10:in `<top (required)>'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `block in load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:258:in `load_dependency'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/railties-5.1.4/lib/rails/engine.rb:549:in `load_seed'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/tasks/database_tasks.rb:270:in `load_seed'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/railties/databases.rake:184:in `block (2 levels) in <top (required)>'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/rake-12.1.0/exe/rake:27:in `<top (required)>'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/bin/ruby_executable_hooks:15:in `eval'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/bin/ruby_executable_hooks:15:in `<main>'
SQLite3::SQLException: table refinery_pages has no column named slug
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/sqlite3-1.3.13/lib/sqlite3/database.rb:91:in `initialize'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/sqlite3-1.3.13/lib/sqlite3/database.rb:91:in `new'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/sqlite3-1.3.13/lib/sqlite3/database.rb:91:in `prepare'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/sqlite3_adapter.rb:212:in `block (2 levels) in exec_query'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies/interlock.rb:46:in `block in permit_concurrent_loads'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/concurrency/share_lock.rb:185:in `yield_shares'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies/interlock.rb:45:in `permit_concurrent_loads'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/sqlite3_adapter.rb:209:in `block in exec_query'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract_adapter.rb:612:in `block (2 levels) in log'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract_adapter.rb:611:in `block in log'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/notifications/instrumenter.rb:21:in `instrument'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract_adapter.rb:603:in `log'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/sqlite3_adapter.rb:208:in `exec_query'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/database_statements.rb:102:in `exec_insert'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/database_statements.rb:133:in `insert'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/query_cache.rb:17:in `insert'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/relation.rb:61:in `insert'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/persistence.rb:589:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/counter_cache.rb:178:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/locking/optimistic.rb:76:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/attribute_methods/dirty.rb:296:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/callbacks.rb:340:in `block in _create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/callbacks.rb:131:in `run_callbacks'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/callbacks.rb:827:in `_run_create_callbacks'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/callbacks.rb:340:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/timestamp.rb:95:in `_create_record'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/persistence.rb:563:in `create_or_update'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/callbacks.rb:336:in `block in create_or_update'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/callbacks.rb:131:in `run_callbacks'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/callbacks.rb:827:in `_run_save_callbacks'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/callbacks.rb:336:in `create_or_update'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/persistence.rb:162:in `save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/validations.rb:50:in `save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/attribute_methods/dirty.rb:43:in `save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/transactions.rb:313:in `block in save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/transactions.rb:384:in `block in with_transaction_returning_status'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/database_statements.rb:235:in `block in transaction'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/transaction.rb:194:in `block in within_new_transaction'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/transaction.rb:191:in `within_new_transaction'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/connection_adapters/abstract/database_statements.rb:235:in `transaction'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/transactions.rb:210:in `transaction'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/transactions.rb:381:in `with_transaction_returning_status'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/transactions.rb:313:in `save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/suppressor.rb:46:in `save!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/persistence.rb:51:in `create!'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/refinerycms-pages-4.0.0/db/seeds.rb:2:in `<top (required)>'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `block in load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:258:in `load_dependency'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/railties-5.1.4/lib/rails/engine.rb:549:in `load_seed'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/railties-5.1.4/lib/rails/railtie.rb:185:in `public_send'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/railties-5.1.4/lib/rails/railtie.rb:185:in `method_missing'
/Users/e832319/work/ruby/rickrockstar/db/seeds.rb:10:in `<top (required)>'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `block in load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:258:in `load_dependency'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activesupport-5.1.4/lib/active_support/dependencies.rb:286:in `load'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/railties-5.1.4/lib/rails/engine.rb:549:in `load_seed'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/tasks/database_tasks.rb:270:in `load_seed'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/activerecord-5.1.4/lib/active_record/railties/databases.rake:184:in `block (2 levels) in <top (required)>'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/gems/rake-12.1.0/exe/rake:27:in `<top (required)>'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/bin/ruby_executable_hooks:15:in `eval'
/Users/e832319/.rvm/gems/ruby-2.4.2@refinery/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => db:seed
```

**Comment by bricesanchez** [comment]
The problem probably stands here : https://github.com/refinery/refinerycms/blob/master/pages/db/seeds.rb#L54

Since the latest version of Globalize, we have dropped columns in the base table which have translation: 
https://github.com/refinery/refinerycms/blob/master/pages/db/migrate/20170703015418_remove_translated_columns_to_refinery_pages.rb

**Comment by ciponthenet** [comment]
Great! That was quick 👍

**Comment by bricesanchez** [comment]
Could you confirm this PR fix your problem #3316
